master/usage/ipv6.md

```diff
@@ -1,20 +1,158 @@
+ensure Kubernetes and Calico work in IPv6. Specifically you may need
+the following to be set.
+
+- Each host must have an IPv6 address that is reachable from the other
+  hosts.
+  > **Note:** Each host running Calico currently must also have an IPv4
+  > address due to [bug 1204](https://github.com/projectcalico/calico/issues/1204).
+  {: .alert .alert-info}
+- The sysctl setting `net.ipv6.conf.all.forwarding` must be enabled by
+  setting it to `1`.  This ensures both Kubernetes service traffic
+  and Calico traffic is forwarded appropriately.
+- Typically it will not be necessary but if hosts do not have a default
+  IPv6 route it is necessary to add one.
+
+### Kubernetes Component Setup
+
+To deploy {{site.prodname}} with IPv6 on Kubernetes it is first necessary
+to setup the Kubernetes components to operate with IPv6. 
+Below are the flags for each Kubernetes component that will help ensure
+IPv6 operation.
+
+- For kube-apiserver:
+  - The `--bind-address` or `--insecure-bind-address` flags should be set
+    to the appropriate IPv6 address or `::` for all IPv6 addresses on
+	the host.
+  - The `--advertise-address` flag should be set to the IPv6 address that
+    nodes should use to access the kube-apiserver.
+  - The `--service-cluster-ip-range` flag should be set to an IPv6 CIDR
+    that will be used for the Service IPs, the DNS service address must
+	be in this range.
+- For kube-controller-manager:
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+  - If the `--allocate-node-cidrs` flag is set then it is necessary to set
+    `--node-cidr-mask-size`. These are necessary when using host-local IPAM,
+    but if using calico-ipam is is easier to remove both flags as they are
+    not needed.
+  - The `--cluster-cidr` flag must be set to match the {{site.prodname}}
+    IPv6 IPPool.
+- For kube-scheduler:
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+- For kubelet:
+  - The `--address` flag should be set to the appropriate IPv6 address
+    or `::` for all IPv6 addresses.
+  - The `--cluster-dns` flag should be set to the IPv6 address that will
+    be used for the service DNS, this must be in the range used for
+	`--service-cluster-ip-range`.
+  - The `--node-ip` flag should be set to the IPv6 address of the node.
+- For kube-proxy:
+  - The `--bind-address` flag should be set to the appropriate IPv6
+    address or `::` for all IPv6 addresses on the host.
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+  - The `--cluster-cidr` flag must be set to match the {{site.prodname}}
+    IPv6 IPPool.
+
+> **Note:** These flags are not the only flags that will need to be set on the
+> kubernetes components but the ones that were found necessary to ensure IPv6
+> operation.
+{: .alert .alert-info}
+
+### Calico Manifest changes
```

Comment:
I'd suggest making this more "verb-y" with a set of steps users can follow.

i.e. 

```
### Enabling IPv6 in Calico

1. Update IPAM to allocate IPv6 addresses

< Example here >

2. Configure Calico with the desired IPv6 IP Pool

< Example here >

3. Enable IPv6 data plane support

 < Example here >

```

Reply:
I thought when we chatted we said a list would be more appropriate because it is just setting these values. If we're looking for a procedure style what about something like:
````
1. Edit your manifest making the following changes.
    <Bullet list>
2. Apply your manifest as you normally would.
```

I'm also not sure about the `<Example here>` pieces as these are changes needed in one large yaml which makes it tricky to take a snippet from.